### PR TITLE
Java March 2020 Release

### PR DESCRIPTION
**Pull request by alzimmermsft**
Contains high level release note for the March 2020 release for Java.

### CODE REVIEW

File: releases/2020-03/java.md [hunk 1]
```diff
@@ -11,18 +11,20 @@ The Azure SDK team is pleased to announce our March 2020 client library releases
...
-- Storage (Blobs, Blobs Batch, Queues, File Shares, DataLake)
+- App Configuration
+- Identity
+- Storage (Blobs, Blobs Batch, Queues, File Shares)
 
 #### Preview
 
 - Cosmos
-- Identity
-- Key Vault (Keys)
```
Comment (g2vinay) [inline releases/2020-03/java.md]: KeyVault (Secrets, Keys, Certificates) previews went out.

File: releases/2020-03/java.md [hunk 2]
```diff
@@ -11,18 +11,20 @@ The Azure SDK team is pleased to announce our March 2020 client library releases
...
+- Search
 - Text Analytics
+- Tracing OpenTelemetry
```
Comment (g2vinay) [inline releases/2020-03/java.md]: Identity preview went out too.

File: releases/2020-03/java.md (hunk 1 above)
Comment (alzimmermsft) [reply releases/2020-03/java.md]: Added

File: releases/2020-03/java.md (hunk 2 above)
Comment (alzimmermsft) [reply releases/2020-03/java.md]: Added

**Comment by alzimmermsft** [comment]
@kyle-patterson is the `azure-sdk - ci` pipeline supposed to be running for this PR? It looks stuck.

**Pull request merged**